example/new-editor-example.js

```diff
@@ -1,4 +1,5 @@
 import wescheme from '../src/languages/wescheme';
+import 'codemirror/lib/codemirror.css';
```

Comment:
Why is this needed? It's already important from within ToggleEditor. I suspect this is leftover from my old `-with-tests` branch, and can be safely deleted